### ISSUE TITLE
Surrogate dashes need to be constructed from nbhy

### ISSUE THREAD

**Issue by cabo** [ietf-tools/xml2rfc#741]
xml2rfc generates surrogate dashes formed by sequences of hyphens.  These are then preferred as line breaking points by HTML renderers such as browsers, breaking e.g. an en-dash by half.

By constructing the surrogate dashes from nbhy characters (except for the last), this confusing effect could be avoided.

(A better solution is to use dashes for dashes, but there has been some resistance to this.)

**Comment by jrlevine** [comment]
I agree that it makes more sense to leave dashes as s in HTML and PDF.
